index.html

```diff
@@ -316,6 +316,58 @@ <h2 class="techies-title">Skills</h2>
     </ul>
 </section>
 
+<!--Contact me section  -->
+<section class="contact-section">
+  <div class="contact-me">
+    <div class="contact-form">
+      <h2 class="con-title">Contact me</h2>
+      <p class="con-paragraph">
+        If you have an application you are interested in developing, a
+        feature that you need built or a project that needs coding. I’d love
+        to help with it
+      </p>
+    <form
+      action="https://formspree.io/f/meqdpjdy"
+      method="POST"
+      class="con-form"
+      id="form"
+    >
+      <input
+        type="text"
+        name="_name"
+        placeholder="Yeremias"
+        maxlength="30"
+        class="form-text"
+        id="fname"
+        required
+      />
+      <input
+        type="email"
+        name="_reply"
+        placeholder="natajayanj14@gmail.com"
+        class="form-text"
+        id="email"
+        required
+      />
+    
+      <textarea
+        name="_message"
+        placeholder="Write your message here"
+        maxlength="500"
+        class="form-text-area"
+        id="textarea"
+        required
+      ></textarea>
+    
+      <li  class="con-but">
+        <button id="submit" type="submit">
+          Get in touch
+        </button>
+      </li>
```

Comment:
- [OPTIONAL] The button is not inside a list and it is not a list item, so the `li` tag is not necessary here. 👍